ci-operator/jobs/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-rhoai-presubmits.yaml

```diff
@@ -6,22 +6,22 @@ presubmits:
     - ^rhoai$
     - ^rhoai-
     cluster: build02
-    context: ci/prow/ci-index
+    context: ci/prow/ci-bundle-opendatahub-operator-bundle
```

Comment:
i dont think we changed way how we build bundle on rhoai branch,
is this needed?

Reply:
The bundle for the rhoai branch has changed. As per our last discussion, I’m no longer building the catalog image; it’s now deployed directly using the operator-sdk command.
```
  - as: opendatahub-operator-bundle
    dockerfile_path: Dockerfiles/bundle.Dockerfile
    skip_building_index: true
```